pvr.vuplus/changelog.txt

```diff
@@ -1,3 +1,12 @@
+v2.4.7 
```

Comment:
I can... Version log is quite inconsistent across commits. Some update it, some do not (specially translation commits)